src/reconcile/rec_visibility.c

```diff
@@ -357,6 +387,58 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
             break;
     }
 
+    /*
```

Comment:
I don't understand much of the change, better to do an online review.

Reply:
Good idea. Unfortunately, this change became more complicated than expected.